drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInViewModel.kt

```diff
@@ -83,6 +84,8 @@ internal class DropInViewModel(
             bundleHandler.paymentMethodsApiResponse = value
         }
 
+    val checkoutSession: CheckoutSession? = bundleHandler.checkoutSession
```

Comment:
This should use a custom getter, because the value in `bundleHandler` could change